geometry_msgs/Point[4] corners
geometry_msgs/Point center
geometry_msgs/Vector3 perturbation_bound
